Formula/scalastyle.rb

```diff
@@ -37,6 +27,8 @@ def main(args: Array[String]) {
         }
       }
     EOS
-    system bin/"scalastyle", testpath/"test.scala"
+    resource("default_config").stage do
```

Comment:
`testpath.install resource("default_config")`

We don't need to stage it in a separate temp directory.